src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -911,6 +908,10 @@ public DesiredActivity getDesiredActivity()
             citizenStatusHandler.setLatestStatus(new TextComponentTranslation("com.minecolonies.coremod.status.waiting"), new TextComponentTranslation("com.minecolonies.coremod.status.rainStop"));
             return DesiredActivity.IDLE;
         }
+        else if (citizenData.isPaused())
+        {
+            return DesiredActivity.IDLE;
```

Comment:
1) you removed mourning above, on purpose?
2) Move this in the AbstractEntityAI class and there go to idle.